package.json

```diff
@@ -150,6 +150,7 @@
   },
   "homepage": "https://github.com/mozillla/addons-frontend#readme",
   "dependencies": {
+    "all": "0.0.0",
```

Comment:
I suspect this is not needed. Could you remove it?

Maybe running `yarn remove all` would work. You should remove the changes in this file and in the `yarn.lock` file.

Reply:
Thanks for pointing that out. fixed it.